BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemState.swift

```diff
@@ -45,6 +45,9 @@ protocol AddEditItemState: Sendable {
     /// The state for guided tour view.
     var guidedTourViewState: GuidedTourViewState { get set }
 
+    /// Whether the user belongs to any organization.
+    var hasOrganizations: Bool { get set }
+
```

Comment:
🎨 From what I can see this depends on the `cipher` and the `ownershipOptions`. Couldn't this be just readonly and calculate the value based off these other properties? Correct me if I'm wrong but something like:
```suggestion
    /// Whether the user belongs to any organization.
    var hasOrganizations: Bool {
        cipher.organizationId != nil || ownershipOptions.contains { !$0.isPersonal }
    }
```
This would make the change much simpler and you avoid having to update the variable from outside the state.